### PR TITLE
fix: fix primevue dialog header icon position

### PR DESCRIPTION
**Pull request by MCatherine1994**
- update primevue dialog header icon to always display at top right
- regroup the dialog styles

### CODE REVIEW

File: style-sheets/primevue-components-overrides.scss [hunk 1]
```diff
@@ -537,32 +537,10 @@ ul#nav li.active a {
...
 }
 
-.#{variables.$primevue-prefix}-dialog
-    .#{variables.$primevue-prefix}dialog-header
-    .#{variables.$primevue-prefix}-dialog-header-icon:focus {
-    box-shadow: none;
-}
-
 .#{variables.$primevue-prefix}-dialog-content p:last-child {
     padding-bottom: 48px;
```
Comment (NickSaglioni) [inline style-sheets/primevue-components-overrides.scss:545]: Should we take the opportunity to change this to rem unit?

File: style-sheets/primevue-components-overrides.scss [hunk 2]
```diff
@@ -574,28 +552,49 @@ ul#nav li.active a {
     border-bottom-left-radius: 0.5rem;
 }
 
-.#{variables.$primevue-prefix}-dialog-footer {
+.#{variables.$primevue-prefix}-dialog
+    .#{variables.$primevue-prefix}-dialog-footer {
+    border-top: none;
     padding-bottom: 64px !important;
```
Comment (NickSaglioni) [inline style-sheets/primevue-components-overrides.scss:558]: Same as above, maybe we could change it to rem instead of px.

**Review by NickSaglioni — CHANGES_REQUESTED** [review]
Just a couple of comments, seems to be good otherwise.

File: style-sheets/primevue-components-overrides.scss (hunk 2 above)
Comment (MCatherine1994) [reply style-sheets/primevue-components-overrides.scss:558]: thanks Nick!!

**Review by NickSaglioni — APPROVED** [review]
All seems good. Thanks.

**Pull request merged**